smart_card_connector_app/build/human_readable_supported_readers.txt

```diff
@@ -416,4 +417,4 @@ Yubico Yubikey NEO OTP+CCID
 Yubico Yubikey NEO OTP+U2F+CCID
 Yubico Yubikey NEO U2F+CCID
 Yubico YubiKey OTP+CCID
-Yubico YubiKey OTP+FIDO+CCID
```

Comment:
Please double-check why this line is displayed in the diff. Is there some missing/extra newline here?